docs/product-assembly_dmn-authoring-guide/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,43 @@
+[id='assembly_dmn-authoring-guide']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Designing decision services using Decision Modeling Notation
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: dmn-authoring-guide
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+Decision Modeling Notation (DMN) is an open standard created by the Object Management Group (OMG). It is used to model decisions based on a specific set of rules or preferences. DMN models diagram business decisions from start to finish, with each decision point drawing logic from custom tables known as decision tables.
+
```

Comment:
Can we clarify this paragraph? 
Something along the lines of "Decisions in DMN are modelled using a graphical language as a diagram of decisions and requirements (the Decision Requirements Graph). etc...." 
It would be good to add a simple DRG example as well.